app/controllers/concerns/fraud_review_concern.rb

```diff
@@ -26,8 +26,9 @@ def handle_fraud_rejection
 
   def in_person_prevent_fraud_redirection?
     IdentityConfig.store.in_person_proofing_enforce_tmx &&
-      !current_user.in_person_enrollment_status.nil? &&
-      current_user.in_person_enrollment_status != 'passed'
+      (!current_user.in_person_enrollment_status.nil? &&
+      current_user.in_person_enrollment_status != 'passed') &&
+    !current_user.double_failed?
```

Comment:
I feel like maybe this red/green section should just live on the User model, rather than containing a confusing chain of operations on the user object.

Reply:
yeh it does feel like a lot of things being chained and it would be cleaner to let this live on the user model.